src/Artist.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+
+const Artist = (props) => {
+    return(
```

Comment:
Please use 2 spaces for indentation instead of 2 spaces. Please refer https://stackoverflow.com/a/38083525